src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function (sv, x) {
+      this.result = sv(this.result, x);
```

Comment:
Use the arithmetic operator with the assignment operator to simplify the expression. Instead of `this.result = sv(this.result, x);`, you can use `this.result = sv.call(this, x);` since the first argument `result` is always `this.result`.